From 36d49e4f5b43a31663a9a919dc3cd860bbdde0e9 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 18 Jan 2021 15:01:57 +0200
Subject: [PATCH] [SW-20810] gaudi: increase sram section

In order to get LZO inside preboot we need to increase .sram section by
4KB.

Change-Id: I4d6d0991614abe2ebd07a5d8dbb3380c540659de
Signed-off-by: Guy Nisan <gnisan@habana.ai>
(cherry picked from commit e460b508abb4422a30b98fb097d7af97304a1c68)
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 3577d00..3869a9b 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -8,7 +8,7 @@
 #define COMMON_GAUDI_H_
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0x10000		/* 64KB */
+#define PREBOOT_SIZE			0x11000		/* 68KB */
 #define SRAM_SIZE			0x1400000	/* 20MB */
 
 #define HL_SP_SRAM_BASE			0x7FFBFE0000
